hack/validate-release-jobs.py

```diff
@@ -80,7 +80,8 @@ def validate_jobs(data, definitions):
 def main(git_repo_path):
     releases = read_release_definitions(os.path.join(git_repo_path, release_definition_path))
     job_data = get_job_data(releases)
-    job_definitions = read_job_definitions(os.path.join(git_repo_path, job_definitions_path))
+    for job_definitions_paths in job_definitions_path:
```

Comment:
The should be swapped around:
`for job_definitions_path in job_definitions_paths:`